Misc/NEWS.d/next/Security/2024-02-18-03-14-40.gh-issue-115398.tzvxH8.rst

```diff
@@ -0,0 +1,8 @@
+Allow controlling Expat >=2.6.0 reparse deferral (CVE-2023-52425) by adding
```

Comment:
```
Allow controlling Expat >=2.6.0 reparse deferral (CVE-2023-52425) by adding
five new methods:

* :meth:`xml.etree.ElementTree.XMLParser.flush`
* :meth:`xml.etree.ElementTree.XMLPullParser.flush`
* :meth:`xml.parsers.expat.xmlparser.GetReparseDeferralEnabled`
* :meth:`xml.parsers.expat.xmlparser.SetReparseDeferralEnabled`
* :meth:`xml.sax.expatreader.ExpatParser.flush`
```

Reply:
@gpshead applied to all backports now:
- 3.8 — https://github.com/python/cpython/pull/116275/commits
- 3.9 — https://github.com/python/cpython/pull/116272/commits
- 3.10 — https://github.com/python/cpython/pull/116270/commits
- 3.11 — https://github.com/python/cpython/pull/116268/commits
- 3.12 — https://github.com/python/cpython/pull/116248/commits